src/main/java/com/ericsson/eiffel/remrem/generate/config/SecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.ericsson.eiffel.remrem.generate.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.ldap.authentication.ad.ActiveDirectoryLdapAuthenticationProvider;
+
+// TODO Uncomment to enable AD backed security
```

Comment:
How is this intended to work? We need to use security if it is specified in configuration. Otherwise without security.